game/scripts/vscripts/util.ts

```diff
@@ -83,9 +83,16 @@ export function getOrError<T>(obj: T | undefined, msg?: string): T {
 }
 
 /**
- * Updates the goal display.
- * @param goals Goals to display in the UI.
+ * Sets whether a player can give orders to any of his controlled units and hero.
+ * @param canIssueOrders Whether the player can now give orders to his controlled units and hero.
  */
+export function setCanPlayerIssueOrders(canIssueOrders: boolean){
+    GameRules.Addon.canPlayerIssueOrders = canIssueOrders;
```

Comment:
I don't think this is a good idea, couldn't you use the section orderFilter instead? That way we control this stuff for each section in the section file instead of messing around with a 'global' that will probably cause a bunch of weird side effects.

Reply:
Resolved in latest commits.